docs/ldml/tr35-keyboards.md

```diff
@@ -187,17 +189,16 @@ Note that in parts of this document, the format `@x` is used to indicate the _at
 
 ### <a name="Compatibility_Notice" href="#Compatibility_Notice">Compatibility Notice</a>
 
-> 👉 Note: CLDR-TC has agreed that the changes required were too extensive to maintain compatibility. For this reason, the DTD used here is _not_ compatible with DTDs from prior versions of CLDR such as v41 and prior.
->
-> To process earlier XML files, use the prior DTD and specification, such as v41 found at <https://www.unicode.org/reports/tr35/tr35-66/tr35.html>
+> 👉 Note: CLDR-TC has agreed that the changes required were too extensive to maintain compatibility. For this reason, the `ldmlKeyboard3.dtd` DTD used here is _not_ compatible with DTDs from prior versions of CLDR such as v43 and prior.
 >
+> To process earlier XML files, use the data and specification from v43.1, found at <https://www.unicode.org/reports/tr35/tr35-69/tr35.html>
 
 ### <a name="Accessibility" href="#Accessibility">Accessibility</a>
 
 Keyboard use can be challenging for individuals with various types of disabilities. For this revision, the committee is not evaluating features or architectural designs for the purpose of improving accessibility. Such consideration could be fruitful for future revisions. However, some points on this topic should be made:
 
 1. Having an industry-wide standard format for keyboards will enable accessibility software to make use of keyboard data with a reduced dependence on platform-specific knowledge.
-2. Some features, such as multiTap and flicks, have the potential to reduce accessibility and thus should be discouraged. For example, multiTap requires pressing keys at a certain speed, and flicks require a more complex movement (press-and-flick) beyond a simple tap. Alternatively, inclusion of accessible methods of generating the same outputs (for example, simple keys on an additional layer), should be considered.
+2. Features which require certain levels of mobility or speed of entry should be considered for their impact on accessibility. This impact could be mitigated by means of additional, accessible methods of generating the same output.
```

Comment:
I am wondering if even this statement is too much. Implementations should be able to synthesize alternate ways of accessing these features with accessibility controls, so IMO it may not be necessary or even desirable to have multiple ways of accessing keys specified in a keyboard layout.

Reply:
This text no longer says "additional methods _such as adding keys to the base layer_." So an additional, accessible method could be synthesized, or it could be in the layout itself.  The point was just to note items tha tmight be of concern.

With that, does it still need revision?

Reply:
I think the mitigation advice should be clearly directed at implementers, not keyboard authors. That'd be enough.